README.md

```diff
@@ -1 +1,106 @@
+
+// Want to set the field to 1:
+client.collections.setInheritance("token", "lpath", 1, OptionalInt.of(1));
+```
+If the operation request includes more than one optional fields, the method will accept a params object that you pass as an 
+instance of a specific class containing those fields. You can set only the desired optional fields as needed. The name 
+of this class follows the pattern `endpointMethodParams`. If you do not need to pass any optional parameters, you can 
+simply pass `null`. For example, the `remove` operation in the collections endpoint has two optional fields: `noTrash` and `recurse`:
+
+
+```java
+// Want to set the noTrash field. Since remove() has more than one optional field, we will need to instantiate a class 
+// for that optional field.
+CollectionsRemoveParams removeParams = new CollectionsRemoveParams();
+removeParams.setNoTrash(1);
+client.collections().remove(token, lpath, removeParams);
+
+// Not setting any optional fields: 
+client.collections().remove(token, lpath, null);
+```
+
+
+
+## Usage Example 
+The following shows a demonstration on how to use the client to create and remove a collection.
+
+```java
+String baseUrl = "http://11.11.111.111:9000/irods-http-api/0.3.0";
+
+// Create an instance of the IrodsHttpClient with the specified base URL.
+IrodsHttpClient client = new IrodsHttpClient(baseUrl);
+
+// Declare a variable to hold the response from the HTTP request. The Response object holds the HTTP status code
+// and the body of the response. Can be retrieved by using .getHttpStatusCode() and .getBody(), respectively.
+Response res;
+
+// Authenticate a rodsadmin user with the username `rods` and password `rodspass`. An authentication request
+// has a response body containing the bearer token that can be used to execute operations as the authenticated user.
+res = client.authenticate("rods", "rodspass");
+String token = res.getBody(); // Set a token variable to the token returned from the request.
+
+// We will create a collection. We don't want to use the createIntermediates field, so we pass it an empty OptionalInt
+String lpath = "/tempZone/home/rods/testCollection";
+res = client.collections().create(token, lpath, OptionalInt.empty());
+if (res.getHttpStatusCode() != 200) {
+    // HTTP request was not successful.
+    System.err.println("Request failed. Unable to create collection. HTTP Status Code: " + res.getHttpStatusCode());
+} else {
+    // Retrieving request was successful so response body will not be null.
+    // Using Jackson to parse the JSON response.
+    ObjectMapper mapper = new ObjectMapper();
+    JsonNode rootNode = null;
+    try {
+        rootNode = mapper.readTree(res.getBody());
+        boolean created = rootNode.path("created").asBoolean();
+        if (created) {
+            System.out.println("Creating collection was a success!");
+        } else {
+            System.err.println("Unable to create a collection");
+        }
```

Comment:
Let's replace these lines with the following:
```java
System.out.println("Created: " + rootNode.path("created").asBoolean());
```

Reply:
Done.